src/NuGet.Core/NuGet.Packaging/Signing/Utility/CertificateChainUtility.cs

```diff
@@ -235,4 +234,4 @@ internal static IEnumerable<string> GetMessagesFromChainStatuses(IEnumerable<X50
                 .OrderBy(x => x, StringComparer.Ordinal);
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
Undo changes to this file.  Exclude this file from your PR.

Reply:
Thanks! Fixed.